cobigen/cobigen-core/src/main/java/com/devonfw/cobigen/impl/config/TemplateSetConfiguration.java

```diff
@@ -0,0 +1,83 @@
+package com.devonfw.cobigen.impl.config;
+
+import java.util.List;
+
+/**
+ * mdukhan This Class is used to save specific properties
+ *
+ */
+public class TemplateSetConfiguration {
+
+  private List<String> groupIds;
+
+  private boolean allowSnapshots;
+
+  private boolean disableLookup;
```

Comment:
the disableLookup should result in groupIds containing/not containing the cobigen-templates group id. Therefore the property isn't necessarry at this point.